src/plugins/opensearch_dashboards_overview/public/components/overview/overview.test.tsx

```diff
@@ -173,33 +173,80 @@ const makeProps = () => ({
 
 describe('Overview', () => {
   describe('renders', () => {
-    it('with solutions and features', () => {
+    it('with news, solutions, and features', () => {
       const props = {
         ...makeProps(),
       };
       const component = shallowWithIntl(<Overview {...props} />);
       expect(component).toMatchSnapshot();
     });
 
-    it('without solutions and with features', () => {
+    it('with news, without solutions, and with features', () => {
       const props = {
         ...makeProps(),
         solutions: [],
       };
       const component = shallowWithIntl(<Overview {...props} />);
       expect(component).toMatchSnapshot();
     });
-    it('with solutions and without features', () => {
+
+    it('with news, with solutions, and without features', () => {
       const props = {
         ...makeProps(),
         features: [],
       };
       const component = shallowWithIntl(<Overview {...props} />);
       expect(component).toMatchSnapshot();
     });
-  });
 
-  // ToDo: Add tests for all the complications of Overview
-  // https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4693
-  it.todo('renders each of the complications of Overview');
+    it('without news, with solutions and features', () => {
+      const props = {
+        ...makeProps(),
+        newsFetchResult: null,
+      };
+      const component = shallowWithIntl(<Overview {...props} />);
+      expect(component).toMatchSnapshot();
```

Comment:
If we were to re-write as a unit test, we'd want to assert that:

1. The `<NewsFeed>` component is not rendered
2. The `.osdOverviewMore` section is rendered (solutions)
3. The `.osdOverviewData` section is not rendered (solutions)

Reply:
Is it worth repurposing this to ensure that newsfeed is not returned? or just delete in the same line as the tutorials.

I think originally we left it in because we could have re-used it. But now it just feels like bloat.

Reply:
@joshuarrrr So should I rewrite those tests to finish this issue or this one will be merged as it is now and I will create a new one and fix it later

Reply:
@thanhinhchtom From my perspective, this is an iterative improvement, and I'd be in favor of merging as-is with the follow-up improvements tracked in a separate issue (and subsequent PR). But other maintainers may have other perspectives.